ingest/scripts/prepare_metadata.py

```diff
@@ -36,18 +36,18 @@ class Config:
 
 def split_authors(authors: str) -> str:
     """Split authors by each second comma, then split by comma and reverse
-    So Xi,L.,Yu,X. becomes L. Xi, X. Yu
+    So Xi,L.,Yu,X. becomes  Xi, L.; Yu, X.
     Where first name and last name are separated by no-break space"""
     single_split = authors.split(",")
     result = []
 
     for i in range(0, len(single_split), 2):
         if i + 1 < len(single_split):
-            result.append(single_split[i + 1].strip() + " " + single_split[i].strip())
+            result.append(single_split[i].strip() + ", " + single_split[i + 1].strip())
         else:
             result.append(single_split[i].strip())
 
-    return ", ".join(result)
+    return "; ".join(result) + ";"
```

Comment:
We can probably omit the trailing `;` - this isn't something that we need in Loculus - we can add it in the ENA submission code if it's needed there - it's also not in the docstring above, would have to be  `Xi, L.; Yu, X.;`

Reply:
I think enforcing this will make it less likely that a user incorrectly just adds 2 names separated by a comma without realizing we have an expected author format, e.g. Anna Parker, Cornelius Roemer would be accepted by us and we would then think it is one name with surname=Anna Parker